Essentials/src/com/earth2me/essentials/perm/impl/SuperpermsHandler.java

```diff
@@ -66,11 +66,13 @@ public boolean tryProvider() {
 
     public String getEnabledPermsPlugin() {
         String enabledPermsPlugin = null;
-        List<String> specialCasePlugins = Arrays.asList("PermissionsEx", "GroupManager",
-                "SimplyPerms", "Privileges", "bPermissions", "zPermissions", "PermissionsBukkit",
-                "DroxPerms", "xPerms");
         for (Plugin plugin : Bukkit.getPluginManager().getPlugins()) {
-            if (specialCasePlugins.contains(plugin.getName())) {
+            final boolean match = plugin.getName().toLowerCase().contains("permissions") ||
+                    plugin.getName().toLowerCase().endsWith("perms") ||
+                    plugin.getName().equals("GroupManager") ||
```

Comment:
I didn't think it was worth it for just two exceptions. It's faster doing it that way too, as opposed to constructing a list each time the method is called. Not a huge deal imho, but I'll change it if that would be preferred. 🙂